src/test/aerospike_scan/scan_basics.c

```diff
@@ -1022,6 +1022,42 @@ TEST(scan_operate, "scan operate")
 	as_record_destroy(rec);
 }
 
+TEST(scan_operate_ttl, "scan operate ttl")
+{
+	as_error err;
+	as_string str;
+	as_string_init(&str, "bar", false);
+	uint32_t ttl = 123456;
+	as_operations ops;
+	as_operations_inita(&ops, 1);
+	ops.ttl = ttl;
```

Comment:
As discussed, using `ops.ttl` only addresses background ops scans/queries, it doesn't address background UDF scans/queries.

Instead of using the ops, I think there should be a TTL on the scan/query policy.

(BTW, did you test with queries as well?)